recipes-core/systemd/files/20-network-io.conf

```diff
@@ -1,7 +1,8 @@
-net.core.wmem_default=524288
```

Comment:
Lets leave the value in the same place, as well if we dont change it.

Reply:
This is the order prefered by the documentations I have found (first default, then max), and also the order when you run sysctl -a.